dashboard/lib/logic/task_grid_filter.dart

```diff
@@ -80,11 +88,35 @@ class TaskGridFilter extends FilterPropertySource {
   RegExp? get hashFilter => _hashProperty.regExp;
   set hashFilter(RegExp? regExp) => _hashProperty.regExp = regExp;
 
-  /// The [showLuci] property is a boolean that indicates whether to display tasks produced
-  /// by a Luci stage in the devicelab.
+  /// The [showWindows] property is a boolean that indicates whether to display tasks produced
+  /// by a Windows stage in the devicelab.
   /// This property will filter out columns on the build dashboard.
-  bool? get showLuci => _luciProperty.value;
-  set showLuci(bool? value) => _luciProperty.value = value;
+  bool? get showWindows => _windowsPorperty.value;
+  set showWindows(bool? value) => _windowsPorperty.value = value;
+
+  /// The [showMac] property is a boolean that indicates whether to display tasks produced
+  /// by a Windows stage in the devicelab.
```

Comment:
nit: Dart docs are expected to be one sentence, empty line, then the rest of the docs.

See https://dart.dev/guides/language/effective-dart/documentation